### PR TITLE
Issue-12:added uuid dependency for Bounty Resource Number creation

### PR DESCRIPTION
**Pull request by Behold3th**
Added uuid dependency through yarn for Bounty Resource Number creation.

### CODE REVIEW

**Comment by amaredeus** [comment]
Hmm I guess we should have posted a CONTRIBUTION.md, generally across open source projects the code merge flows from

main <- develop <- feature/name-of-feature

Here is a good read on this

https://nvie.com/posts/a-successful-git-branching-model/

And there's a good read on git tooling on chapter 3 https://git-scm.com/book/en/v2 but basically we should create a PR from feature/12 -> develop. 

Let me know if you have any questions!

**Review by amaredeus — CHANGES_REQUESTED** [review]
Please do 

```
git pull origin develop
```

**Review by amaredeus — CHANGES_REQUESTED** [review]
I think the last change is select "develop"
chrome_2021-06-21_20-41-53

**Comment by Behold3th** [comment]
> I think the last change is select "develop"
> chrome_2021-06-21_20-41-53

should be all set :) thank you!

**Review by amaredeus — APPROVED** [review]

**Comment by amaredeus** [comment]
Thank you! Everything looks good, feel free to squash and merge

File: package.json [hunk 1]
```diff
@@ -31,6 +31,11 @@
     ]
   },
   "dependencies": {
+    "dotenv": "^10.0.0",
```
Comment (amaredeus) [inline package.json:34]: I'm reading up on next.js and it looks this Next.js includes this, so we can remove this https://nextjs.org/docs/basic-features/environment-variables

**Review by amaredeus — CHANGES_REQUESTED** [review]

**Pull request merged**